bcda/bcdacli/cli_test.go

```diff
@@ -836,3 +834,110 @@ func getRandomPort(t *testing.T) int {
 
 	return listener.Addr().(*net.TCPAddr).Port
 }
+
+func CreateTestZipFile(file string) error {
```

Comment:
Do you see value in having this available outside of the cli package? If not, I think we should not export it and make it createTestZipFile.

Reply:
Nit - should we move these helper functions beneath the two test functions?